package.json

```diff
@@ -113,6 +113,7 @@
     "test-integration-mysql": "DIALECT=mysql npm run test-integration",
     "test-integration-postgres": "DIALECT=postgres npm run test-integration",
     "test-integration-postgres-native": "DIALECT=postgres-native npm run test-integration",
+    "test-integration-query-interface-postgres": "DIALECT=postgres ./node_modules/.bin/mocha --globals setImmediate,clearImmediate --ui tdd --check-leaks --colors -t 15000 --reporter spec 'test/integration/dialects/postgres/query-interface.test.js'",
```

Comment:
We don't need to add separate test command for `query interface`, we have integration test commands that will run those tests.